helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -36,57 +41,76 @@ spec:
         class: MEDIUM
         service: kubelet
       record: raw_slo_errors
-    - expr: sum(raw_slo_requests) by (service, cluster_type, cluster_id)
-      record: slo_requests
-    - expr: sum(raw_slo_errors) by (service, cluster_type, cluster_id)
-      record: slo_errors
-    - expr: sum(sum_over_time(raw_slo_errors[5m])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[5m])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate5m
-    - expr: sum(sum_over_time(raw_slo_errors[30m])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[30m])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate30m
-    - expr: sum(sum_over_time(raw_slo_errors[1h])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[1h])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate1h
-    - expr: sum(sum_over_time(raw_slo_errors[6h])) by (cluster_type, cluster_id, service, class) / sum(sum_over_time(raw_slo_requests[6h])) by (cluster_type, cluster_id, service, class)
-      record: slo_errors_per_request:ratio_rate6h
-  - name: service-level-apps
-    rules:
+    - expr: "scalar(vector(0.999))"
+      labels:
+        class: MEDIUM
+        service: kubelet
+      record: slo_target
     # record when pods of a daemonset with label "label_giantswarm_io_monitoring_basic_sli" are down
-    - record: monitoring:managed_apps:number_of_pods_unavailable
-      expr: |
+    - expr: |
         kube_daemonset_status_number_unavailable
         and on(daemonset,cluster_id,cluster_type,namespace)
         kube_daemonset_labels{label_giantswarm_io_monitoring_basic_sli='true'}
+      labels:
+        class: MEDIUM
+        service: managed-apps
+      record: raw_slo_errors
     # record when pods of a deployment with label "label_giantswarm_io_monitoring_basic_sli" are down
-    - record: monitoring:managed_apps:number_of_pods_unavailable
-      expr: |
+    - expr: |
         kube_deployment_status_replicas_unavailable
         and on(deployment,cluster_id,cluster_type,namespace) 
         kube_deployment_labels{label_giantswarm_io_monitoring_basic_sli='true'}
+      labels:
+        class: MEDIUM
+        service: managed-apps
+      record: raw_slo_errors
```

Comment:
`raw_slo_errors` for managed apps are the same metrics as recorded before

Reply:
Is it ok to create recording rule with the same name and labels twice?

Reply:
I think so. These metrics were called `monitoring:managed_apps:number_of_pods_unavailable` before and defined in exactly the same way.

Reply:
How does that work? Does one overwrite the other? I don't get how you could have multiple time series with the exact same labels

Reply:
They don't have the same labels. Only the first one has `daemonset` label key, the 2nd one `deployment` label key.

Reply:
How I understand it the difference is the `workload_type` label from the original series. It can be `deployment`, `daemonset` or `statefulset`.

Reply:
Alright so the labels defined there are extra labels on top of what the underlying metric already has. All good then.